delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/files/Files.scala

```diff
@@ -696,11 +697,12 @@ object Files {
   ): Task[Files] = {
     val idAvailability: IdAvailability[ResourceAlreadyExists] =
       (project, id) => resourceIdCheck.isAvailableOr(project, id)(ResourceAlreadyExists(id, project))
-    apply(config, eventLog, acls, orgs, projects, storages, idAvailability)
+    apply(config, storageTypeConfig, eventLog, acls, orgs, projects, storages, idAvailability)
   }
 
   private[files] def apply(
       config: FilesConfig,
+      storageTypeConfig: StorageTypeConfig,
```

Comment:
If you need this on the implicit side, and since this is already a `private` scoped method, you could have it placed on the implicits block, right?